tests/InputFiles/GrMhd/GhValenciaDivClean/BinaryNeutronStar.yaml

```diff
@@ -81,6 +81,10 @@ VariableFixing:
     MaxDensityCutoff: 1.0e-08
     LorentzFactorCap: 10.0
 
+PrimitiveFromConservative:
+  CutoffDForInversion: 2.0e-12
```

Comment:
Should these match something in `VariableFixing`? E.g. `CutoffD` and `DensityCutoff`? If so, maybe use the `*density_cutoff` and similarly define a `&cutoff_d` and `*density_cutoff` to guarantee things stay in sync? Same of the other input files.